linetools/isgm/abssurvey.py

```diff
@@ -253,7 +251,7 @@ def ions(self, iZion, skip_null=False):
         Table of values for the Survey
         """
         keys = [u'name', ] + self.abs_sys()[0]._ionN.keys()
-        t = copy.deepcopy(self.abs_sys()[0]._ionN[0:1])
+        t = Table(copy.deepcopy(self.abs_sys()[0]._ionN[0:1]))  # Avoids mixin trouble
```

Comment:
I never really know what is going on with deepcopy and classes (!). It would be nice to make a `copy` method for any class object we end up needing deepcopy for.